### PR TITLE
feat(Knob): update TS structure

### PR DESCRIPTION
**Pull request by soumyaloka**
## Description

This PR updates Knob type declarations

## References

LUI-954

### CODE REVIEW

**Review by arwehrman — APPROVED** [review]

File: packages/@lightningjs/ui-components/src/components/Knob/Knob.d.ts [hunk 1]
```diff
@@ -18,17 +18,19 @@
 
 import lng from '@lightningjs/core';
 import Base from '../Base';
-import type { Color, StylePartial } from '../../types/lui';
+import { Color, StylePartial } from '../../types/lui';
 
-export type KnobStyle = {
+type KnobStyle = {
   circleColor: Color;
```
Comment (anthony9187) [inline packages/@lightningjs/ui-components/src/components/Knob/Knob.d.ts:24]: is this right? circleColor only appears in tones/modes, so would it be accessible via `Knob.style.circleColor`?

Comment (soumyaloka) [reply packages/@lightningjs/ui-components/src/components/Knob/Knob.d.ts:24]: Yes I think circleColor property is accessible via Knob.style.circleColor, it is used in slider component.

File: packages/@lightningjs/ui-components/src/components/Knob/Knob.d.ts [hunk 2]
```diff
@@ -18,17 +18,19 @@
...
   radius: lng.Tools.CornerRadius;
 };
```
Comment (arwehrman) [inline packages/@lightningjs/ui-components/src/components/Knob/Knob.d.ts:27]: I think you still need to add the namespace and then update the class with the namespace but should check with Anthony
```
declare namespace Knob {
  export type TemplateSpec = Base.TemplateSpec;
}
```

Comment (anthony9187) [reply packages/@lightningjs/ui-components/src/components/Knob/Knob.d.ts:27]: nah in cases where we aren't making changes to the TemplateSpec or TypeConfig we should omit them and just pass the parent's interfaces

**Review by anthony9187 — APPROVED** [review]

**Pull request merged**